test/Bootstrapper.FunctionalTests/BootstrapperTests.cs

```diff
@@ -114,7 +114,7 @@ public void BootstrapperShowsVersionAndReturnsZeroExitCodeWhenVersionOptionWasGi
 
         [Theory]
         [MemberData(nameof(RuntimeComponents))]
-        public void BootstrapperInvokesApplicationHostWithInferredAppBase_ProjectDirAsArgument(string flavor, string os, string architecture)
+        public void BootstrapperInvokesApplicationHostWithExplicitAppBase(string flavor, string os, string architecture)
```

Comment:
We have these cases:

`dnx run` - ln. 148
`dnx --appbase` - ln. 128
`dnx . run` - https://github.com/aspnet/dnx/blob/dev/test/Bootstrapper.FunctionalTests/BootstrapperTests.cs#L264

Let me repeat after @anurse - 'What is `--project`?" (and how my change is affecting it - btw. all the tests passed so I assume I did not break it)